client/src/pages/OnboardingPage/OnboardingPage.css

```diff
@@ -0,0 +1,75 @@
+.OnboardingPage {
+  font-family: var(--default-font);
+  background: linear-gradient(180deg, rgba(255, 255, 255, 0) 72.4%, rgba(255, 255, 255, 0.75) 100%),
+    #ffcb08;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+}
+.OnboardingPage__wrapper {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.OnboardingPage .Logo {
+  width: 100%;
+  height: 16.5rem;
+  background-size: contain;
+}
+.OnboardingPage__carousel-dots-container {
+  margin: 0 auto;
+}
+.OnboardingPage__form {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
```

Comment:
You have this class defined at least 3 times - can you define it once and reuse it? Please go through and reduce any other duplicated classes